Frontend/research/src/Actions/Index.js

```diff
@@ -87,42 +88,65 @@ export const getCategory = () => dispatch => {
 }
 
 
-// Add a new link
+// ADD LINK
 export const ADD_LINK = "ADD_LINK";
 export const ADD_LINK_SUCCESS = "ADD_LINK_SUCCESS";
 export const ADD_LINK_FAILURE = "ADD_LINK_FAILURE";
 
 export const addLink = link => dispatch => {
+    console.log(link)
     dispatch({ type: ADD_LINK })
-    return axios.post('https://rticle.herokuapp.com/api/user/articles', link, {headers: {Authorization: localStorage.getItem('token')}})
+    return axios.post('https://rticle.herokuapp.com/api/user/articles', link, { headers: { Authorization: localStorage.getItem('token') } })
```

Comment:
you can also set the localStorage.getItem("token") to its one variable since you have to use it may times in the file